### DISCUSSION TITLE
Placeholder-text on Textedit

### DISCUSSION THREAD

**Discussion by develcooking** [slint-ui/slint#5229] [Q&A] [answered]
Hi, I want to use the TextEdit widget with a placeholder text. Is it possible to achieve this with a workaround or through a proper implementation?

LineEdit has a placeholder-text propertie while Textedit doesn't have it

This missing feature isn't tracked in issue https://github.com/slint-ui/slint/issues/474

**Comment by ghost** [comment, marked as answer]
Hi, yeah there is a proper way to implement it for the TextEdit. I have planned to take over some of the widget issues this week. I will take this also to my agenda. Thanks for the request.

**Comment by ghost** [comment]
This PR is a preparation for the `placeholder` on `TextEdit` #5236

**Reply by ghost** [reply]
The PR for the `placeholder-text` #5239

**Reply by ghost** [reply]
PR is merged.
